templates/travel/partials/results/results.snip.html

```diff
@@ -0,0 +1,128 @@
+<!-- Results -->
+<section class="travel-results pb1 md-pt1">
+  <div class="max-width-3 mx-auto px1 md-px2">
+    <amp-list
+      class="travel-block-list travel-results-list"
+      layout="flex-item"
+      {{> ../util/amp-list-props.snip.html}}
+    >
+      <template type="amp-mustache">
+        {{=[[ ]]=}}
+        <div class="flex flex-wrap mxn1">
+          {{#results.length}}
+            {{#results}}
+            <div class="col-12 sm-col-6 lg-col-4 p1 travel-results-result">
+              <div class="relative travel-results-result">
+                <a class="travel-results-result-link" href="#">
+                  <amp-img
```

Comment:
all the amp-img tags should have an alt attribute, needs to be accessible (i think we did not look at this in the other PRs as well), would appreciate if you can take a quick pass and add an alt in a separate PR for the other images